File: festivalC#/festivalc/model/Spectacol.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace festivalc.model
{
    public class Spectacol
    {
        private int id;
        private int idA;
        private string numeArtist;
        private string date;
        private string ora;
        private string locatie;
        private int locuridisponibile;
        private int locuriocupate;

        public Spectacol(int id, int idA, string date, string ora, string locatie, int nrlocuri, int locuriocupate)
        {
            this.id = id;
            this.idA = idA;
            this.date = date;
            this.ora = ora;
            this.locatie = locatie;
            this.locuridisponibile = nrlocuri;
            this.locuriocupate = locuriocupate;
        }

        public int Id {
            get { return id; }
            set { id = value; }
        }
        public int IdA{
            get { return idA; }
            set { idA = value; }
        }

        public string NumeArtist
        {
            get { return numeArtist; }
            set { numeArtist = value; }
        }

        public string Date {
            get { return date; }
            set { date = value; }
        }

        public string Ora
        {
            get { return ora; }
            set { ora = value; }
        }

        public string Locatie {
            get { return locatie; }
            set { locatie = value; }
        }
        public int Locuridisponibile {
            get { return locuridisponibile; }
            set { locuridisponibile = value; }
        }
        public int Locuriocupate {
            get { return locuriocupate; }
            set { locuriocupate = value; }
        }
    }
}

File: festival/src/main/java/model/Spectacol.java
package model;


public class Spectacol implements HasId<Integer> {
    private Integer id;
    private Integer idA;
    private String numeArtist;
    private String date;
    private String ora;
    private String locatie;
    private Integer locuridisponibile;
    private Integer locuriocupate;

    public Spectacol(Integer id, Integer idA, String date, String ora, String locatie, Integer locuridisponibile, Integer locuriocupate) {
        this.id = id;
        this.idA = idA;
        this.date = date;
        this.ora = ora;
        this.locatie = locatie;
        this.locuridisponibile = locuridisponibile;
        this.locuriocupate = locuriocupate;
    }

    @Override
    public Integer getId() {
        return id;
    }

    @Override
    public void setId(Integer id) {
        this.id = id;
    }

    public String getOra() {
        return ora;
    }

    public void setOra(String ora) {
        this.ora = ora;
    }

    public String getNumeArtist() {
        return numeArtist;
    }

    public void setNumeArtist(String numeArtist) {
        this.numeArtist = numeArtist;
    }

    public Integer getIdA() {
        return idA;
    }

    public void setIdA(Integer idA) {
        this.idA = idA;
    }

    public String getDate() {
        return date;
    }

    public void setDate(String date) {
        this.date = date;
    }

    public String getLocatie() {
        return locatie;
    }

    public void setLocatie(String locatie) {
        this.locatie = locatie;
    }

    public Integer getLocuridisponibile() {
        return locuridisponibile;
    }

    public void setLocuridisponibile(Integer locuridisponibile) {
        this.locuridisponibile = locuridisponibile;
    }

    public Integer getLocuriocupate() {
        return locuriocupate;
    }

    public void setLocuriocupate(Integer locuriocupate) {
        this.locuriocupate = locuriocupate;
    }
}
